nnvm/python/nnvm/frontend/onnx.py

```diff
@@ -754,7 +771,7 @@ def _get_convert_map(opset):
         'Cast': Cast.get_converter(opset),
         'Reshape': Reshape.get_converter(opset),
         'Concat': Renamer('concatenate'),
-        'Split': AttrCvt('split', {'split': 'indices_or_sections'}),
+        'Split': Split.get_converter(opset),
```

Comment:
Pls add a testcase.

Reply:
ONNX Split:  https://github.com/onnx/onnx/blob/master/docs/Operators.md#Split
TVM split: https://docs.tvm.ai/nnvm_top.html#nnvm.symbol.split

We need a converter to convert ```split``` from onnx to ```indices_or_sections``` from tvm.
I don't know what's test case do you need. But I had test it in my onnx model and working well.

Reply:
Good that you tested with your model and it's working well.

Please refer to https://github.com/dmlc/tvm/blob/master/nnvm/tests/python/frontend/onnx/test_forward.py for existing testcases for ONNX.

You could use the split layer attributes from your model for the test case here.

Reply:
Add test case done.